### PR TITLE
Add unit tests for dropwizard.util.StringsTest

### PR DESCRIPTION
**Pull request by paulkennethkent**
###### Problem:
I've analysed your code base and noticed that `dropwizard.util.StringsTest` is not fully tested.

###### Solution:
I've written some tests for the functions in this class with the help of [Diffblue](https://www.diffblue.com/) [Cover](https://www.diffblue.com/products). Hopefully, these tests should help you detect any regressions caused by future code changes. If you would find it useful to have additional tests written for this repository, I would be more than happy to look at other particular classes that you consider important.

### CODE REVIEW

File: dropwizard-util/pom.xml [hunk 1]
```diff
@@ -45,5 +45,10 @@
             <artifactId>jackson-databind</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>junit</groupId>
```
Comment (joschi) [inline dropwizard-util/pom.xml]: Please use JUnit 5 for writing tests. The dependency is already included in "test" scope for all sub-modules.

Comment (jplock) [reply dropwizard-util/pom.xml]: Also assertj for assertions

**Comment by paulkennethkent** [comment]
Thanks for the feedback,  the tests have now been modified to use JUnit 5 and AssertJ libraries.

**Comment by joschi** [comment]
@paulkennethkent Thanks!

**Pull request merged**